assets/src/edit-story/components/form/dateTime/timePicker.js

```diff
@@ -0,0 +1,367 @@
+// eslint-disable-next-line eslint-comments/disable-enable-pair
```

Comment:
I know it's a big component, but some unit tests would be awesome here!

Reply:
Note that the whole datepicker stuff is ported over from Gutenberg and should be removed again after alpha. Doesn't really make sense to add tests for it under these circumstances.